app/docs/0.10.x/admin-api.md

```diff
@@ -1438,5 +1439,67 @@ HTTP 200 OK
 }
 ```
 
+---
+
+### List active targets
+
+Retrieve a list of active targets (targets whose most recent weight is not 0)
+under a given upstream.
```

Comment:
"...for a given upstream" (instead of "under") - not sure which term is more typical for us.